pkgs/development/tools/rust/cargo-cross/docker-Add-Nix-Store-volume-support.patch

```diff
@@ -0,0 +1,42 @@
+From e86ad2e5a55218395df7eaaf91900e22b809083c Mon Sep 17 00:00:00 2001
```

Comment:
Did you upstream this patch? Do you plan to do it? If so we could use fetchpatch instead.

Reply:
Yes, I did. Let me find how to do this.